tests/forte/advanced_pipeline_test.py

```diff
@@ -280,5 +280,76 @@ def tearDown(self) -> None:
             os.remove(self._pl_config_path)
 
 
+class InferringOntologyTreeAutomaticallyTst(unittest.TestCase):
+    """
+    Test the Inferring Ontology Tree Automatically by using Recursive Method
+    """
+
+    def setUp(self) -> None:
+        self.expected = [
+            {
+                "entry_name": "ft.onto.wikipedia.WikiInfoBoxProperty",
+                "parent_entry": "forte.data.ontology.top.Generics",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiInfoBoxMapped",
+                "parent_entry": "forte.data.ontology.top.Generics",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiCategory",
+                "parent_entry": "forte.data.ontology.top.Generics",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiPage",
+                "parent_entry": "forte.data.ontology.top.Annotation",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiBody",
+                "parent_entry": "forte.data.ontology.top.Annotation",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiSection",
+                "parent_entry": "forte.data.ontology.top.Annotation",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiParagraph",
+                "parent_entry": "forte.data.ontology.top.Annotation",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiTitle",
+                "parent_entry": "forte.data.ontology.top.Annotation",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiArticleTitle",
+                "parent_entry": "forte.data.ontology.top.Annotation",
+            },
+            {
+                "entry_name": "ft.onto.wikipedia.WikiAnchor",
+                "parent_entry": "forte.data.ontology.top.Annotation",
+            },
+        ]
+
+        self.pipeline_1: Pipeline = Pipeline()
+        from ft.onto.wikipedia import WikiPage
+
+        self.pipeline_2: Pipeline = Pipeline()
```

Comment:
I did not understand the purpose of this test. Why does pipeline_1 and pipeline_2 be different?